packages/grouping/src/js/grouping.js

```diff
@@ -747,8 +747,9 @@
        * @param {Grid} grid grid object
        * @param {GridColumn} column the column we want to aggregate
        * @param {string} aggregationType of the recognised types from uiGridGroupingConstants or one of the custom aggregations from gridOptions
+       * @param {string} aggregationLabel to be used instead of the default label. If empty string is passed, label is omitted
```

Comment:
The intention is to be able to pass any label and have each row show that label. For our purposes, we just so happen to want to hide the aggregation labels but for other users this would hopefully allow them to use count and have the label say "Sum" instead.